src/main/scala/nl.knaw.dans.easy.download/EasyDownloadServlet.scala

```diff
@@ -51,7 +51,7 @@ class EasyDownloadServlet(app: EasyDownloadApp) extends ScalatraServlet
     (getUUID, getPath) match {
       case (Success(uuid), Success(Some(path))) => download(authRequest, userName, uuid, path)
       case (Success(_), Success(None)) => BadRequest("file path is empty")
-      case (Failure(t), _) => BadRequest(t.getMessage) // invalid uuid
+      case (Failure(t), _) => NotFound(t.getMessage) // invalid uuid
```

Comment:
Maybe this is out of scope for this issue, but actually, I am not quite sure why a failure from `getPath` is considered a Bad Request. It is not clear to me how `getPath` could actually fail, either.